scripts/provision/provision.sh

```diff
@@ -0,0 +1,250 @@
+#!/bin/bash
+#
+# Copyright (c) 2018
+# Mainflux
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+###
+# Core Metadata
+###
+printf "> Provisioning Addressibles\n"
+
+# Addressable for the Device Service
+curl http://localhost:48081/api/v1/addressable -X POST -s -S -d @- <<EOF
+{
+    "name":"camera control",
+    "protocol":"HTTP",
+    "address":"172.17.0.1",
+    "port":49977,
+    "path":"/cameracontrol",
+    "publisher":"none",
+    "user":"none",
+    "password":"none",
+    "topic":"none"
+}
+EOF
+
+echo ""
+
+# Addressable for the Device (the camera in this case)
+curl http://localhost:48081/api/v1/addressable -X POST -s -S -d @- <<EOF
+{
+    "name":"camera1 address",
+    "protocol":"HTTP",
+    "address":"172.17.0.1",
+    "port":49999,
+    "path":"/camera1",
+    "publisher":"none",
+    "user":"none",
+    "password":"none",
+    "topic":"none"
+}
+EOF
+
+###
+# Core Data
+###
+printf "\n> Provisioning Value Descriptors\n"
```

Comment:
If edgexfoundry/device-sdk-go/example/simple is used, then there is no need to add value descriptors